gramps/gui/plug/_windows.py

```diff
@@ -1135,13 +1135,17 @@ def __init__(self, addon_update_list, parent_window):
                 pos = iter
         if pos:
             self.list.selection.select_iter(pos)
-        self.window.run()
 
-    def close(self, widget):
+        self.show()
+
+    def build_menu_names(self, obj):
+        return (self.title, " ")
+
+    def close_btn(self, widget):
```

Comment:
Yes it does.  See [line 569](https://github.com/gramps-project/gramps/blob/master/gramps/gui/managedwindow.py#L569).